pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -2179,6 +2191,7 @@ PyObject *objToJSON(PyObject *Py_UNUSED(self), PyObject *args,
         Object_endTypeContext,
         Object_getStringValue,
         Object_getLongValue,
+        Object_getBigNumStringValue,
```

Comment:
This line throws a compiler error:
```
pandas/_libs/src/ujson/python/objToJSON.c:2194:9: error: initialization of ‘JSINT32 (*)(void *, JSONTypeContext *)’ {aka ‘int (*)(void *, struct __JSONTypeContext *)’} from incompatible pointer type ‘const char * (*)(void *, JSONTypeContext *, size_t *)’ {aka ‘const char * (*)(void *, struct __JSONTypeContext *, long unsigned int *)’} [-Werror=incompatible-pointer-types]
```

Reply:
You will need to update the PyEncoder struct to have the appropriate member definitions